Fast Pair documentation update bundle (#15953)
nrfconnect/sdk-nrf: doc/nrf/external_comp/bt_fast_pair.rst

****************************************

The specific use case of the Google Fast Pair application is indicated by the chosen device type in the Google Nearby Devices console (see the :ref:`ug_bt_fast_pair_provisioning_register_device_type` subsection).

> The `Fast Pair Device Feature Requirements`_ category in the official `Fast Pair`_ documentation defines additional requirements for each supported use case.

Reviewer: ```suggestion
In the official `Fast Pair`_ documentation, the `Fast Pair Device Feature Requirements`_ category defines additional requirements for each supported use case, and specifies a list of mandatory, optional, and unsupported Fast Pair features.
```

> The `Fast Pair Device Feature Requirements`_ category in the official `Fast Pair`_ documentation defines additional requirements for each of the supported use cases.

Reviewer: ```suggestion
The `Fast Pair Device Feature Requirements`_ category in the official `Fast Pair`_ documentation defines additional requirements for each supported use case.
```

> Each listed use case specifies a list of mandatory and unsupported Fast Pair features.

Reviewer 2: `mandatory, optional and not supported.` (e.g. see `Automotive`)

> Each listed use case specifies a list of mandatory, optional and unsupported Fast Pair features.

Reviewer: ```suggestion
```

> If your product is targeting one of the listed use case, you should align your accessory firmware to satisfy its requirements.

Reviewer: ```suggestion
If your product is targeting one of the listed use cases, you must align your accessory firmware to meet these requirements.
```

The locator tags can use different wireless technologies like GPS, Bluetooth LE or UWB for location tracking.
It is even possible to combine multiple technologies in a single product to improve the user experience.

The `Fast Pair Device Feature Requirements for Locator Tags`_ documentation defines the Fast Pair requirements for the locator tag use case.

> If your product is targeting the locator tag use cases, you must configure your application according to these requirements.

Reviewer 2: `locator tag use case`?

Author: this was just a copy past of another sentence so I am going to align also that one
